### PR TITLE
feat(cce): partly fix prePaid node pool

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
due to API problem, we can't delete node pool directly with terraform, so test case is unavailable for now.
```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccSomethingV0_basic'
...
=== RUN   TestAccSomethingV0_basic
--- PASS: TestAccSomethingV0_basic (70.75s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       70.796s
```

### CODE REVIEW

File: docs/resources/cce_node_pool.md [hunk 1]
```diff
@@ -39,6 +39,44 @@ resource "huaweicloud_cce_node_pool" "node_pool" {
...
+    volumetype = "SAS"
+  }
+  data_volumes {
+    size       = 100
+    volumetype = "SAS"
+  }
+}
+```
+
+  -> You need to remove all nodes in the node pool on the console, before deleting a prepaid node pool.
```
Comment (niuzhenguo) [inline docs/resources/cce_node_pool.md]: use ~> instead to be more notable.

Comment (Jason-Zhang9309) [reply docs/resources/cce_node_pool.md]: sure

**Pull request merged**